src/main/java/com/softserve/actent/model/dto/equipment/EquipmentCreateDto.java

```diff
@@ -15,11 +15,11 @@
     private String description;
 
     @NonNull
-    private boolean satisfied;
+    private Boolean satisfied;
```

Comment:
why you use Class instead of primitive?

Reply:
Because wrappers not auto-setting values(for ex. int - 0, boolean will be false); if client didn't sent anything, value will be null.